index.html

```diff
@@ -84,5 +84,224 @@
+        </div>
+
+        <a href="#" class="card1-project">See Project</a>
+      </div>
+      <!-- End of Card 2 -->
+
+      <!-- Card 3 -->
+      <div class="card ">
+        <div>
+          <img src="images/part/snapshootportfolio3.png" alt="Tonic" />
+        </div>
+
+        <div class="card1paragraph-text">
+          <h2>Tonic</h2>
+        </div>
+
+        <div class="card1description-text">
+          <p class="canopy marginright">CANOPY</p>
+
+          <img src="images/part/counter.png" alt="counter" class="counterimg marginright" />
+          <p class="imgtext marginright">Back End Dev</p>
+
+          <img src="images/part/counter.png" alt="counter" class="counterimg marginright" />
+          <p class="imgtext marginright">2015</p>
+        </div>
+
+        <p class="cardpara">
+          A daily selection of privately personalized reads; no accounts or sign-ups required.
+        </p>
+
+        <div class="card1-button">
+          <p class="card1-button-paragraph">html</p>
+          <p class="card1-button-paragraph">css</p>
+          <p class="card1-button-paragraph">javascript</p>
+        </div>
+
+        <a href="#" class="card1-project">See Project</a>
+      </div>
+      <!-- End of Card 3 -->
+
+      <!-- Card 4 -->
+      <div class="card ">
+        <div>
+          <img src="images/part/snapshootportfolio4.png" alt="Tonic" />
+        </div>
+
+        <div class="card1paragraph-text">
+          <h2>Multi-Post Stories</h2>
+        </div>
+
+        <div class="card1description-text">
+          <p class="canopy marginright">CANOPY</p>
+
+          <img src="images/part/counter.png" alt="counter" class="counterimg marginright" />
+          <p class="imgtext marginright">Back End Dev</p>
+
+          <img src="images/part/counter.png" alt="counter" class="counterimg marginright" />
+          <p class="imgtext marginright">2015</p>
+        </div>
+
+        <p class="cardpara">
+          A daily selection of privately personalized reads; no accounts or sign-ups required.
+        </p>
+
+        <div class="card1-button">
+          <p class="card1-button-paragraph">html</p>
+          <p class="card1-button-paragraph">css</p>
+          <p class="card1-button-paragraph">javascript</p>
+        </div>
+
+        <a href="#" class="card1-project">See Project</a>
+      </div>
+      <!-- End of Card 4 -->
+    </section>
+
+    <!-- End of Work section -->
+
+    <!-- About me section -->
+    <section class="aboutsection">
+      <div class="aboutparagraph">
+        <h2 class="name">About</h2>
+        <h2 class="name">Myself</h2>
+
+        <p class="about">
+          I’m a software developer! I can help you build a product , feature or website Look through
+          some of my work and experience! If you like what you see and have a project you need
+          coded, don’t hestiate to contact me.
+        </p>
+
+        <p class="connect">Let's connect</p>
```

Comment:
This should be an `hx` tag. If you notice from the design that it looks quite bigger than a normal `p` element. You can use `h5` or `h6` instead and style it accordingly

Reply:
Okay, i will look into this